src/Settings.php

```diff
@@ -145,6 +145,11 @@ public static function parseArguments(array $arguments)
         $arguments = new ArrayIterator(array_slice($arguments, 1));
         $settings = new self;
 
+        // Use the currently invoked php as the default if possible
+        if (defined(PHP_BINARY)) {
```

Comment:
`defined` takes string with a constant name as an argument, so here it checks if a constant with name `/usr/bin/php` or alike (`PHP_BINARY`'s value) is defined which would most likely be false, so `$settings->phpExecutable` will likely never be set. Shouldn't it be `defined('PHP_BINARY')`?

Reply:
True! @exussum12 Can you fix it?